pixeltable/type_system.py

```diff
@@ -11,6 +11,7 @@
 import av
 import numpy as np
 import PIL.Image
+import pyarrow
```

Comment:
Another import we don't want at the top level. Dot-completion shouldn't matter here, plus all the pyarrow functions are going away anyway with Oscar's changes.

Reply:
Ok, done.